Sources/TuistLoader/Loaders/CachedManifestLoader.swift

```diff
@@ -22,10 +22,10 @@ public class CachedManifestLoader: ManifestLoading {
     private let tuistVersion: String
     private let decoder = JSONDecoder()
     private let encoder = JSONEncoder()
-    private var helpersCache: [AbsolutePath: String?] = [:]
-    private var pluginsCache: [AbsolutePath: String?] = [:]
-    private var cacheDirectory: AbsolutePath!
-    private var plugins: Plugins?
+    @Atomic private var helpersCache: [AbsolutePath: String?] = [:]
+    @Atomic private var pluginsCache: [AbsolutePath: String?] = [:]
+    @Atomic private var cacheDirectory: AbsolutePath!
+    @Atomic private var plugins: Plugins?
```

Comment:
@luispadron out of curiosity why do we need to keep this? I see in other case we use the `helpersDirectoryLocator.locate(at: path)` for helps, can't we do something similar for plugins?

Reply:
As commented offline with @luispadron, plugin hash is actually projects-independent, so we will save the hash on register directly, simplify the implementation.